ui/tests/acceptance/dashboard-test.js

```diff
@@ -197,39 +198,36 @@ module('Acceptance | landing page dashboard', function (hooks) {
         .hasText('http://127.0.0.1:8200');
       assert.dom(SELECTORS.vaultConfigurationCard.configDetailsField('default_lease_ttl')).hasText('0');
       assert.dom(SELECTORS.vaultConfigurationCard.configDetailsField('max_lease_ttl')).hasText('2 days');
-      assert.dom(SELECTORS.vaultConfigurationCard.configDetailsField('tls_disable')).hasText('Enabled');
```

Comment:
`tls_disable` was set to `true` but the assertion was looking for `Enabled`.

Reply:
Thanks for catching that Jordan!